package.json

```diff
@@ -89,5 +90,30 @@
 		"version_replace_paths": [
 			"src"
 		]
+	},
+	"bundlewatch": {
+		"files": [
+			{
+				"path": "./js/build/*.js",
+				"maxSize": "2 kB"
+			},
+			{
+				"path": "./js/build/index.js",
+				"maxSize": "1 mB"
+			},
+			{
+				"path": "./js/build/*.css",
+				"maxSize": "1 kB"
+			},
+			{
+				"path": "./js/build/index.css",
+				"maxSize": "8 kB"
+			},
+			{
+				"path": "./google-listings-and-ads.zip",
+				"maxSize": "12 mB",
+				"compression": "none"
+			}
+		]
```

Comment:
💅  Personally I would prefer to put these into a separate file like `.bundlewatch.config.js` and have bundlewatch to _automatically_ read the file, unfortunately currently bundlewatch requires us to specify it in the command line like `bundlewatch --config .bundlewatch.config.js` (see the [documentation](https://bundlewatch.io/#/getting-started/using-a-config-file)), which is an unnecessary friction in my opinion. Hopefully they can support that in the future.

So with that said, good to have this `bundlewatch` section here in this `package.json` so that we can just run `npx bundlewatch`. 👍